src/Config.php

```diff
@@ -14,6 +14,7 @@
 
 namespace PhpCsFixer;
 
+use PhpCsFixer\ConfigInterface;
```

Comment:
It's in the same namespace, shouldn't need to be imported.

Reply:
Sorry, I was configuring php-cs-fixer on Visual Studio Code with Intelphpense, something wrong must have happened because nothing was working until I added this import. I found other errors and since it's fixed this error is not happening. You can forget this PR. Sorry